### DISCUSSION TITLE
Professional Firmware Compatibility with Other Ender 3 Series Printers.

### DISCUSSION THREAD

**Discussion by Broadmeadow-Tekton** [mriscoc/Ender3V2S1#1177] [Q&A]
Is the Professional Firmware for the Creality 3D Ender V2 / S1 compatible with the 3 Pro by chance?  If not, what would have to be changed on the 3 Pro to make the firmware compatible?

Thanks.

**Comment by mriscoc** [comment]
Check the requirements: https://github.com/mriscoc/Ender3V2S1/wiki#requirements

**Reply by csanii123** [reply]
I posted a question in the feedback section of the Dec2023 edition asking about the Ender3 (not Pro) screen and a 4.2.7 board.....I posted a few weeks ago....could you please answer ?  Userid csanii123 on Dec9th 
Thx.
